span-processing-config-service-impl/src/main/java/org/hypertrace/span/processing/config/service/SpanProcessingConfigServiceImpl.java

```diff
@@ -43,21 +52,49 @@
 @Slf4j
 class SpanProcessingConfigServiceImpl
     extends SpanProcessingConfigServiceGrpc.SpanProcessingConfigServiceImplBase {
+
+  private static final String SPAN_PROCESSING_CONFIG_SERVICE = "span.processing.config.service";
+  private static final String SYSTEM_LEVEL_EXCLUDE_SPAN_RULES = "system.level.exclude.span.rules";
+
   private final SpanProcessingConfigRequestValidator validator;
   private final ExcludeSpanRulesConfigStore excludeSpanRulesConfigStore;
   private final TimestampConverter timestampConverter;
   private final ApiNamingRulesManager apiNamingRulesManager;
+  private List<ExcludeSpanRule> systemLevelExcludeSpanRules;
```

Comment:
Do we need this, if we already have the map?

Reply:
Right. Updated to use the map in other usages as well.